particles/Tutorial/Javascript/Demo/source/TicTacToeBoard.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+/* global defineParticle */
+
+defineParticle(({SimpleParticle, html}) => {
+
+const template = html`
+  <style>
+    .grid-container {
+      display: grid;
+      grid-template-columns: 50px 50px 50px;
+      grid-column-gap: 0px;
+    }
+
+    .valid-butt {
+      border: 1px outset blue;
+      height: 50px;
+      width: 50px;
+      cursor: pointer;
+      background-color: lightblue;
+    }
+
+    .invalid-butt {
+      border: 1px outset blue;
+      height: 50px;
+      width: 50px;
+      cursor: pointer;
```

Comment:
maybe: `cursor: not-allowed;`

or even: `pointer-events: none;`

The latter would not allow a click at all, as in: it would not cause invocation of the handler in JS.